content/legal/licenses.rst

```diff
@@ -1,8 +1,105 @@
-:nosearch:
```

Comment:
Why?

Reply:
Because licenses.rst referenced licenses/licenses.rst, and I'm moving the content of licenses/licenses into licenses

Reply:
Oops, didn't pay attention ^^